frontend/src/pages/Landing.css

```diff
@@ -0,0 +1,59 @@
+@media (max-width: 2560px) {
+  .hero-container {
+    background-image: url("../assets/background_desktop.png");
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+    width: "100%";
```

Comment:
Size units aren't in strings for css files. They're only in strings for React files